app/views/home/index.html.erb

```diff
@@ -1,14 +1,22 @@
-<% if @job_postings&.any? %>
+<% if @job_postings.any? %>
   <ul class="divide-y divide-slate-100 w-full">
-  <% @job_postings.each do |job| %>    
-    <li class="flex items-start gap-4 px-4 py-3 border-b border-slate-100">
-    <div class="flex flex-col gap-0 min-h-[2rem] items-start justify-center w-full">
-      <h4 class="w-full text-base text-slate-700"><%= job.title %></h4>
-      <p class="w-full text-sm text-slate-500"><%= job.company_profile.name %></p>
-    </div>
-  </li>
-  <% end %>
+    <% @job_postings.each do |job| %>    
+      <li class="flex items-start gap-4 px-4 py-3 border-b border-slate-100">
+        <div class="overflow-hidden bg-white rounded shadow-md text-slate-500 shadow-slate-200">
+          <div class="p-6">
+            <h4 class="w-full text-base text-slate-700"><%= link_to job.title, job_posting_path(job) %></h4>
+            <p><%= job.job_type.name %></p>
+            <p><%= job.company_profile.name %></p>
+          </div>
+        </div>
+      </li>
+    <% end %>
   </ul>
+<% elsif @search_error %>
```

Comment:
That's why I don't like to mix index and search results 😂 

@matheusfsantana 

I'll create a new issue so we can split it later.